doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -206,10 +206,13 @@ In order for the storage solution(s) to be conformant with the Reference Archite
 
 |Ref|Specification|Details|Requirement Trace|
 |---|---|---|---|
-|`ra2.stg.001`||||
-|`ra2.stg.002`||||
-|`ra2.stg.003`||||
-|`ra2.stg.004`||||
+|`ra2.stg.001`| Ephemeral Storage | An implementation must support ephemeral storage, for the unpacked container images to be stored and executed from, as a directory in the filesystem on the worker node on which the container is running. <br>See the [Container runtimes](#4.4) section above for more information on how this meets the requirement for ephemeral storage for containers. ||
+|`ra2.stg.002`| Kubernetes Volumes | An implementation may attach additional storage to containers using Kubernetes Volumes. ||
+|`ra2.stg.003`| Kubernetes Volumes | An implementation may use Volume Plugins (see `ra2.stg.005` below) to allow the use of a storage protocol (e.g. iSCSI, NFS) or management API (e.g. Cinder, EBS) for the attaching and mounting of storage into a Pod. ||
+|`ra2.stg.004`| Persistent Volumes | An implementation may support Kubernetes Persistent Volumes (PV) to provide persistent storage for Pods (requirement 'req.inf.stg.01'.<br>Persistent Volumes exist independent of the lifecycle of containers and/or pods. |[req.inf.stg.01](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter02.md#23-kubernetes-architecture-requirements)|
```

Comment:
Suggest change to relative link.
```suggestion
|`ra2.stg.004`| Persistent Volumes | An implementation may support Kubernetes Persistent Volumes (PV) to provide persistent storage for Pods (requirement 'req.inf.stg.01'.<br>Persistent Volumes exist independent of the lifecycle of containers and/or pods. |[req.inf.stg.01](./chapter02.md#23-kubernetes-architecture-requirements)|
```

Reply:
@tomkivlin for readdocs (I maybe wrong) but I think we need the absolute url

Reply:
> @tomkivlin for readdocs, I maybe wrong, I think we need the absolute url

Ah OK my bad, I hadn't realised that change.  Will resolve and approve.